cosmos/precompile/governance/governance.go

```diff
@@ -215,7 +216,7 @@ func (c *Contract) GetProposal(
 	return c.getProposalHelper(ctx, proposalID)
 }
 
-// GetProposals is the method for the `getProposal` method of the governance precompile contract.
+// `GetProposals` is the method for the `getProposal` method of the governance precompile contract.
```

Comment:
remove quotes everywhere, we realised this was incorrect

Reply:
makes sense